### PR TITLE
Change Symbols to Strings for input data

### LINKED ISSUE #705: Use strings instead of Symbols in the create_input_dataframes function
(this means Tulipa will now expect strings instead of Symbols)

### PR DESCRIPTION
**Pull request by datejada**
# Pull request details

## Describe the changes made in this pull request

To use DuckDB types

## List of related issues or pull requests

Closes #705 

## Collaboration confirmation

As a contributor I confirm

-   [x] I read and followed the instructions in README.dev.md
-   [x] The documentation is up to date with the changes introduced in this Pull Request (or NA)
-   [ ] Tests are passing
-   [x] Lint is passing

### CODE REVIEW

File: test/test-io.jl [hunk 1]
```diff
@@ -135,7 +135,7 @@ end
             println(io, line)
         end
     end
-    missing_asset = Symbol(split(lines[end], ",")[1]) # The asset the was not included
+    missing_asset = split(lines[end], ",")[1] # The asset the was not included
```
Comment (abelsiqueira) [inline test/test-io.jl:138]:
```suggestion
    missing_asset = split(lines[end], ",")[1] # The asset that was not included
```

**Review by abelsiqueira — APPROVED** [review]
Found a small fix in the comment, but it's otherwise good to go.

**Comment by datejada** [comment]
@abelsiqueira I have mixed feelings with the benchmark result, any comments?

**Comment by abelsiqueira** [comment]
Oof, don't like it. The slowdown is too large for something that we thought would be ~0% change. I will create a revert PR so we can see if there is a similar increase or if the benchmarks are broken. If there is a speed up, I think we might need to investigate before some other changes.

**Pull request merged**